tests/utils/utility.py

```diff
@@ -0,0 +1,31 @@
+import contextlib
```

Comment:
I think it's fine if you want to toss this in tests/utils/__init__.py.  utils/utility.py is kind of awkward.